### DISCUSSION TITLE
Allowing creators to use custom A-Frame components inside of Spoke on Hubs Cloud

### DISCUSSION THREAD

**Discussion by traviskaufman** [mozilla/hubs#3870] [Hubs Cloud Questions] [answered]
Hey all, I have a hubs cloud deployment running on AWS and would like to extend Hubs such that our creators can create rooms that include some custom A-Frame components my team develops. 

I've read the guides on customizing the hubs client and read the guide on Spoke. Where would be the best to start if I wanted to accomplish something like this?

Thank you so much for your help! ❤️

**Comment by wmurphyrd** [comment]
We're introducing this in the [Immers Space](https://immers.space) hubs fork with a feature called `spoke-tagger` that lets creators enable custom components on objects in the scene by using the name field in Spoke. Check out the diff on this PR: immers-space/hubs#11

**Reply by robinkwilson** [reply]
Super neat! Thanks for the link @wmurphyrd !!

**Comment by robinkwilson** [comment, marked as answer]
Hello! This issue about deploying a custom spoke client answers your question: https://github.com/mozilla/Spoke/issues/1006

**Comment by robinkwilson** [comment]
@traviskaufman I'm not sure which answer helped you the most, can you mark the answer of your choosing please! :)

**Reply by traviskaufman** [reply]
Yes, of course! Sorry still getting used to GH discussions 😄 Thanks so much again for your help.
